pipeline/utils.py

```diff
@@ -54,3 +62,12 @@ def relpath(path, start=posixpath.curdir):
     if not rel_list:
         return posixpath.curdir
     return posixpath.join(*rel_list)
+
+
+def set_std_streams_blocking():
```

Comment:
It would be nice if this had a docstring explaining what it does and why it's needed.